repo/format/format_manager.go

```diff
@@ -80,7 +80,7 @@ func (m *Manager) maybeRefreshNotLocked() error {
 	}
 
 	// current format not valid anymore, kick off a refresh
-	return m.refresh(m.ctx)
+	return m.refresh(ctx)
```

Comment:
It seems that one of the primary changes here is to stop using a `ctx` field in the Manager and instead use a context that is passed in. Most (if not all) of the changes can be traced to wiring the context to `content.Manager.maybeRefreshNotLocked`
Is that right?